ngraph/python/tests/test_onnx/test_onnx_import.py

```diff
@@ -31,8 +31,7 @@ def test_import_onnx_function():
     ie = IECore()
     ie_network = ie.read_network(model=model_path)
 
-    capsule = ie_network._get_function_capsule()
-    ng_function = Function.from_capsule(capsule)
+    ng_function = ng.get_function(ie_network)
```

Comment:
In C++ CNNNetwork contains `get_function` method which returns nGraph function. Current approach is not align with C++ API. 

@akuporos Can we add `get_function` method to CNNNetwork in IE python API?

Something like:
```
ng_function = ie_network.get_function()
```

Reply:
Discussed offline.

Reply:
We discussed it inside the ticket №35348.